features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -149,6 +150,29 @@ private fun RoomMemberList(
     }
 }
 
+@Composable
+private fun RoomMemberSearchResult(
```

Comment:
I think you wanted to rename the method but the previous one is still present

Reply:
Would we have a single method with a param for the header title?

Reply:
Refactored